porchlightapi/settings.py

```diff
@@ -35,5 +36,14 @@
                                'PORCHLIGHT_JSON_FILE',
                                PORCHLIGHT_JSON_FILE_DEFAULT)
 
+PORCHLIGHT_GITHUB_AUTH_DEFAULT = None
+PORCHLIGHT_GITHUB_AUTH = getattr(settings,
+                                 'PORCHLIGHT_GITHUB_AUTH',
+                                 PORCHLIGHT_GITHUB_AUTH_DEFAULT)
+
+PORCHLIGHT_GITHUB_TAG_PATTERN_DEFAULT = r'^v[.0-9]+'
```

Comment:
I am probably not qualified to answer that particular question, but I'm thinking _no_. Ask someone else more involved in DevOps about that.